lib/arisaid/core_ext/hash.rb

```diff
@@ -8,4 +8,8 @@ def stringify_keys
     each_key { |k| result[k.to_s] = self[k] }
     result
   end
+
+  def symbolize_keys
+    self.map{|k,v| [k.to_sym, v] }.to_h
```

Comment:
```suggestion
    self.map { |k,v| [k.to_sym, v] }.to_h
```

既存のコードの書き方を踏襲しておくと無難そう :memo: